pkg/pods/pod.go

```diff
@@ -529,6 +529,12 @@ func (pod *Pod) setupConfig(manifest Manifest, launchables []launch.Launchable)
 	}
 
 	for _, launchable := range launchables {
+		// we need to remove any unset env vars
```

Comment:
Ah, because we re-use env directories between pods. Could you append "... from a previous pod" to the comment?